src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+
+  margin: auto;
+
+  padding: $day-padding;
+  width: getCalendarWidth(
+    $day-size,
+    $day-gap,
+    $day-padding);
+
```

Comment:
what about height? right now your calendar dropping off the viewport)

https://user-images.githubusercontent.com/79811705/186211522-2d77e317-ac8e-4312-9a82-406441ea5ca9.mp4